src/main/java/seedu/address/logic/parser/HelpCommandParser.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.parser;
+
+import seedu.address.logic.commands.HelpCommand;
+import seedu.address.logic.commands.HelpConsultationCommand;
+import seedu.address.logic.commands.HelpLabCommand;
+import seedu.address.logic.commands.HelpTutorialCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new HelpCommand, HelpTutorialCommand,
+ * HelpLabCommand or HelpConsultationCommand object
+ */
+public class HelpCommandParser {
+    /**
+     * Parses the given {@code String} of arguments in the context of the HelpCommand
+     * and returns an HelpCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+
+    public HelpCommand parse(String args) {
```

Comment:
Would it be better to make it a public static method for now? It'd be a bit troublesome to call `HelpCommandParser ().parse(args)` each time!

Reply:
I am not sure about the help commands. @nerdyboy98 wrote them. I just merged them in